### ISSUE TITLE
[1.16.5] Mobs always spawn with dungeons armor

### ISSUE THREAD

**Issue by voxel-ate** [Thelnfamous1/Dungeons-Gear#223]
dungeons_gear-1.16.5-3.2.7 - Zombies and skeletons that are spawned always have dungeons armor, is this a glitch or intentional?
I only have dungeons gear, dungeons library, curios and geckolib installed

![2023-02-20_13 10 24](https://user-images.githubusercontent.com/101897255/220118608-d9b3a06b-a1b9-4e11-8e14-17d78b8e712e.png)

**Comment by aarter180** [comment]
This also occurs in my instance on the same version as well and I am also unsure if this is intentional or not.  It also happens regardless of whether the difficulty is set to easy or hard.

**Comment by aarter180** [comment]
I think this may be a dungeons library issue instead.  When I disabled elite mobs in the dungeons library config, all the vanilla mobs spawned afterwards were armorless.  I also found that the mountaineers from dungeons mobs are also affected by this.  If elite mobs are enabled, only the different colored variants of the mountaineer which also carried a large backpack spawned.  When it is disabled, only the basic brown mountaineer spawned.  Notably, I have never changed the elite mobs chance from its default value 0.15.  But these results make it seem like its acting like the value is 1.00.  And I am using the latest version of the dungeons suite mods in 1.16.
